### PR TITLE
APPEAL-3419

### PR DESCRIPTION
**Pull request by zurbergram**
Resolves [APPEALS-3419](https://vajira.max.gov/browse/APPEALS-3419)

### Description
Work is behind the feature toggle `split_appeal_workflow` which will be disabled in all environments

### Acceptance Criteria
- [ ] Code compiles correctly

### CODE REVIEW

File: client/app/intake/pages/addIssues.jsx [hunk 1]
```diff
@@ -260,17 +267,47 @@ class AddIssuesPage extends React.Component {
       intakeData.addedIssues, intakeData.originalIssues
     );
 
-    const addIssueButton = () => {
+    const buttonConditionalRendering = () => {
```
Comment (zurbergram) [inline client/app/intake/pages/addIssues.jsx:270]: This is missing the feature toggle check

File: app/controllers/split_appeal_controller.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class SplitAppealController < ApplicationController
+  protect_from_forgery with: :exception
+
+  def split_appeal
```
Comment (zurbergram) [inline app/controllers/split_appeal_controller.rb:6]: Add a toggle check here that just returns a 501

File: app/policies/appeal_request_issues_policy.rb [hunk 3]
```diff
@@ -24,6 +25,10 @@ def editable_by_cavc_team_member?
       appeal.tasks.open.where(type: :CavcTask).any?
   end
 
+  def editable_by_ssc_team_member?
+    SupervisorySeniorCouncil.singleton.users.include?(user)
```
Comment (zurbergram) [inline app/policies/appeal_request_issues_policy.rb:29]: Needs a feature toggle is enabled check

File: app/views/appeals/edit.html.erb [hunk 4]
```diff
@@ -2,6 +2,11 @@
   <%= react_component("IntakeEdit", props: {
     userDisplayName: current_user.display_name,
     userCanWithdrawIssues: current_user.can_withdraw_issues?,
+    userCanSplitAppeal: current_user.can_split_appeal?(appeal),
+    appeal: appeal,
```
Comment (zurbergram) [inline app/views/appeals/edit.html.erb:6]: Is the entire appeal needed?

File: client/app/intakeEdit/IntakeEditFrame.jsx [hunk 5]
```diff
@@ -9,32 +9,58 @@ import Link from '@department-of-veterans-affairs/caseflow-frontend-toolkit/comp
...
-  displayClearedEpMessage = (details) => {
+export const StateContext = createContext({});
+
+export const Provider = ({ children }) => {
+  const [reason, setReason] = useState(null);
+  const [otherReason, setOtherReason] = useState('');
+  const [selectedIssues, setSelectedIssues] = useState({});
+
+  return (
+    <StateContext.Provider value={{
```
Comment (zurbergram) [inline client/app/intakeEdit/IntakeEditFrame.jsx:36]: Can the surrounding changes be behind the feature toggle?

File: client/app/intakeEdit/IntakeEditFrame.jsx [hunk 6]
```diff
@@ -70,42 +96,44 @@ export class IntakeEditFrame extends React.PureComponent {
...
+            accentColor: LOGO_COLORS.INTAKE.ACCENT,
+            overlapColor: LOGO_COLORS.INTAKE.OVERLAP,
+          }}
+          userDisplayName={props.userDisplayName}
+          dropdownUrls={props.dropdownUrls}
+          topMessage={topMessage}
+          defaultUrl="/"
+        >
+          <AppFrame>
+            <Route exact path={PAGE_PATHS.CREATE_SPLIT} component={SplitAppealProgressBar} />
```
Comment (zurbergram) [inline client/app/intakeEdit/IntakeEditFrame.jsx:134]: Need feature toggle check

File: client/app/queue/CaseDetailsView.jsx [hunk 7]
```diff
@@ -212,8 +212,32 @@ export const CaseDetailsView = (props) => {
...
+  const splitStorage = localStorage.getItem('SplitAppealSuccess');
+
+  localStorage.removeItem('SplitAppealSuccess');
+
+  // if null, leave null, if true, check if value is true with reg expression.
+  const splitAppealSuccess = (splitStorage === null ? null : (/true/i).test(splitStorage));
+
   return (
     <React.Fragment>
+      {splitAppealSuccess && (
```
Comment (zurbergram) [inline client/app/queue/CaseDetailsView.jsx:225]: Feature toggle check?

File: client/app/intake/pages/addIssues.jsx (hunk 1 above)
Comment (zurbergram) [reply client/app/intake/pages/addIssues.jsx:270]: @cacevesva

**Review by cacevesva — APPROVED** [review]

**Pull request merged**